dail/new-hire-ndnh.vbs

```diff
@@ -240,6 +241,7 @@ IF match_answer_droplist = "NO-RUN NEW HIRE" THEN
 	    	Dialog dialog1
 	    	cancel_confirmation
 	    	MAXIS_dialog_navigation
+			IF worker_signature = "" THEN err_msg = err_msg & vbCr & "* Please sign your case note."
```

Comment:
Err_msg isn't set in the do...loop. No msgbox code to give the worker_signature error message.